### ISSUE TITLE
Badge-title not mobile friendly

### ISSUE THREAD

**Issue by zoglo** [contao/contao#7362]
### Affected version(s)

`4.13.x`, `5.3.x`

### Description

Whilst working on another PR, I've noticed that "long" badge-titles aren't really working on mobile:

Contao `4.13` has the same issue but wouldn't occur unless you have a longer badge-title.
_(Similar approach of fixing them has been done here: https://github.com/contao/contao-manager/pull/819, this wouldn't work with the current contao-backend tho)_

#### Contao `4.13` on a `375px` width device:
![image](https://github.com/user-attachments/assets/e2672816-300f-4700-a169-972f7f130e70)

#### Contao `5.3` on a `375px` width device:
![image](https://github.com/user-attachments/assets/41749e48-9d83-4515-b686-07e868d73644)
As seen in the image above, the newly introduced icons (theme toggle, favorites), make sure that this badge is shown behind it.

### Proposal

I'd not think too much about handling lifestory badge-titles and would propose to let the badge title look like this `<600px`:
![image](https://github.com/user-attachments/assets/36990372-4c5a-413f-8d4d-4f9ba97382ef)

![image](https://github.com/user-attachments/assets/20cd2f26-bada-42cb-8874-076c2b0fcf54)

I could open PRs against `4.13`, `5.3` and `5.4` if needed and if the proposed solution is okay for everyone.

**Comment by zoglo** [comment]
@leofeyer wdyt about the proposed solution? I could provide PRs

**Comment by leofeyer** [comment]
I never use the feature, so I don‘t know what would be the best (in terms of most unobtrusive) way to fix it. Maybe someone who uses it can suggest something?

**Comment by m-vo** [comment]
I used badge titles to differentiate between environments (test/dev/prod…) and I like @zoglo's solution. :+1: If you are using that feature, I would assume you really want to be aware of the title, so the placement up front seems logical to me.

We should make sure the bar is only displayed if there is a badge title set, though.

**Comment by zoglo** [comment]
> We should make sure the bar is only displayed if there is a badge title set, though.

The bar is only displayed when a badge-title exists and only looks like that <600px ;)

**Comment by zoglo** [comment]
Can do a PR for 4.13 - the ones for 5.3 and 5.4 however would be dependent on #7337

**Comment by zoglo** [comment]
See #8267
